packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -374,6 +550,7 @@ const ComboBox = React.forwardRef((props, ref) => {
                     translateWithId={translateWithId}
                     disabled={disabled || readOnly}
                     onClearSelection={handleSelectionClear}
+                    selectionCount={0}
```

Comment:
Could you explain this addition? Not sure why the selectionCount would need to be defined at 0 for this case.

Reply:
So `selectionCount` is a required property and the component now throws a ts error without it. The reason for it being 0, is that the selection count is used to show a badge next to the component when multiple items are selected. In this case, the combobox only allows a single selection, so that badge shouldn't be present and setting it to 0 (rather than 1) accomplishes that.